portfolio/src/main/webapp/script.js

```diff
@@ -107,4 +107,12 @@ function handleSubmit(e) {
   $(this).find('input,textarea').val('');
 }
 
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
nit: You can replace the `document.getElementById('map')` with `$('#map')[0]` as per https://learn.jquery.com/using-jquery-core/faq/how-do-i-pull-a-native-dom-element-from-a-jquery-object/

Reply:
Done.